qiskit/_pubsub.py

```diff
@@ -0,0 +1,126 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2017, IBM.
+#
+# This source code is licensed under the Apache License, Version 2.0 found in
+# the LICENSE.txt file in the root directory of this source tree.
+
+"""
+Message broker for the Publisher / Subscriber mechanism
+"""
+
+from ._qiskiterror import QISKitError
+
+
+class _Broker(object):
+    """ The event/message broker. It's a singleton."""
```

Comment:
Also minor - can you  do a round of removing the space after the `"""` in docstrings?